src/ResponseFactory.php

```diff
@@ -48,6 +49,10 @@ public function getVersion()
 
     public function render($component, $props = [])
     {
+        if ($props instanceof JsonResource) {
```

Comment:
So in my case, I would be passing an array of props, with some being an instance of `JsonResource`.

It would need to iterate each of the props to resolve them as well. It may be worth rethinking this a little...